app/src/main/java/com/kahuis/repairtool/presentation/login/LoginScreen.kt

```diff
@@ -63,43 +61,34 @@ fun LoginView() {
         }
 
         //Get username AND password from loginPackage
-        val uName = singleLineInputLabel("Gebruikersnaam")
+        uName = singleLineInputLabel("Gebruikersnaam")
         Spacer(modifier = Modifier.padding(8.dp))
         val pWord = singelLinePasswordLabel("Wachtwoord")
 
         Spacer(modifier = Modifier.padding(top = 30.dp))
 
         //Login button AND forgot password button
+        /*
+        Op dit moment moet de navigatie en code die hier voor gebruikt wordt nog gemaakt worden.
+        Inloggen kan door op de knop 'inloggen' te klikken zonder username of wachtwoord te gebruiken.
+
+        Dit is bij wijze van test even gedaan om 1 view te maken (die van de klusjesman)
+
+        TODO navigatie zo maken dat username en wachtwoord weer nodig zijn
```

Comment:
If you're adding a TODO to the code, it might be a god practice to create a new ticket with the objective. You can then add this ticket number to the TODO statement: `TODO(RT-201): navigatie zo maken dat username en wachtwoord weer nodig zijn`